astro/src/content/blog/i-just-want-authentication-to-work.mdx

```diff
@@ -0,0 +1,71 @@
+* Consider that when Better Napster, wait I mean your app starts to grow, you will want an Authentication provider that can handle different forms of Authentication such as mobile, desktop and web.
+* Develop on a local instance or self contained envioronment that you can host yourself without risking data corruption in production.
+
+## What Should You Care About If Not Authentication
+
+Now that you have authentication figured out you can start to focus on the real requirements of building an application...
+
+Deciding the rest of your software:
+
+- Framework - [Remix](https://remix.run/), [Next.js](https://nextjs.org/), [SvelteKit](https://kit.svelte.dev/), [Pheonix Framework](https://www.phoenixframework.org/), [Laravel](https://laravel.com/), [Flask](https://flask.palletsprojects.com/en/3.0.x/), [Angular](https://angular.io/)
+- Database - [PlanetScale](https://planetscale.com/), [AlloyDB for PostgreSQL](https://cloud.google.com/alloydb), [AWS Aurora Serverless](https://aws.amazon.com/rds/aurora/serverless/), [CockroachDB Serverless](https://www.cockroachlabs.com/blog/announcing-cockroachdb-serverless/), [EdgeDB](https://www.edgedb.com/), [Neon](https://neon.tech/), [Supabase](https://supabase.com/), [Timescale](https://www.timescale.com/), [Upstash](https://upstash.com/), [Grafbase](https://grafbase.com/), [Turso](https://turso.tech/), [Fauna](https://fauna.com/), [Ably](https://ably.com), [Convex](https://www.convex.dev/), [Hasura](https://hasura.io/), [liveblocks](https://liveblocks.io/)
+- Hosting - [Vercel](https://vercel.com/), [Netlify](https://www.netlify.com/), [Fly.io](https://fly.io/), [Cloudflare](https://www.cloudflare.com/)
+- CMS: [Contentful](https://www.contentful.com/), [Quintype](https://www.quintype.com/), [Hygraph](https://hygraph.com/), [Ghost](https://ghost.org/publishers/), [Strapi](https://strapi.io/), [Sanity](https://www.sanity.io/), [Kontent](https://kontent.ai/), [Optimizely](https://www.optimizely.com/products/orchestrate/content-management/)
+
+There are full backend solutions out there like [Firebase](https://firebase.google.com/), [Supabase](https://supabase.com/), and [Appwrite](https://www.appwrite.io). However, if you don't like some part of this stack then it becomes very difficult for you to decouple your application(s). In this video our Founder and CEO talks about [Authentication as a Microservice](https://youtu.be/SLc3cTlypwM?si=qFzJaeA9NTeXGSfz&t=834) and how to use a JWT in your APIs.
+
+## Finally
+
+Okay now that I got that out of my system I am off to listen to [Connor Price](https://youtu.be/UekfKh59KOw?si=xTM0powkicUHG2Yo) and login to Amazon using my [face](https://www.aboutamazon.com/news/retail/amazon-passwordless-sign-in-passkey), because it is 2023 people!
```

Comment:
```suggestion
Okay now that I got that out of my system I am off to listen to [Connor Price](https://youtu.be/UekfKh59KOw?si=xTM0powkicUHG2Yo) and log in to Amazon using my [face](https://www.aboutamazon.com/news/retail/amazon-passwordless-sign-in-passkey), because it is 2023 people!
```

Verb is `log in`. Noun is `login`.